test/local_integration_test.py

```diff
@@ -121,9 +121,8 @@ def _test_other_endpoints(self):
     def _test_manifest(self):
         manifest_filter = json.dumps({'project': {'is': [self.test_name]}})
         for format_, validator in [
-            (None, self.check_manifest),
```

Comment:
This can't be right. The default value for the format parameter is `tsv` and the integration test should cover that. The title of the commit that introduces this deletion (`Fix: None format is not recognize for generating a manifest`) is also misleading as it claims to be a fix which it is not. At best, it is a work-around, albeit a incorrect one, if I am not mistaken.